### ISSUE TITLE
[Question] What would you need to tweak to simulate cells from a bulk dataset

### ISSUE THREAD

**Issue by Hoohm** [Oshlack/splatter#53]
Hello,
I'd like to simulate a single cell dataset from a bulk data set.
Would you suggest anything special compared to using a single cell dataset as a base?

**Comment by lazappi** [comment]
Hi @Hoohm 

The estimation functions in Splatter assume that you have a single-cell dataset with a fairly large number of samples so they try to do things like fit distributions that don't necessarily work with the number of samples in a typical bulk dataset. For the Splat simulation this is mostly a problem for things like the library size that are fit across samples. The other simulations might have similar problems or could just fail entirely.

In short Splatter isn't really designed for bulk experiments, it can't hurt to give it a go but no guarantee it will work well. If you just want a fairly standard bulk simulation there are other functions around such as the one in the compcodeR package https://bioconductor.org/packages/release/bioc/html/compcodeR.html. 

Hope that helps

**Comment by Hoohm** [comment]
Hey @lazappi,
thanks for the quick response.

I still might give it a go. Just to be sure I understood the paper, the simulation for gene expression are based on "average" values across the reference dataset. This means that Splatter doesn't keep the "real" gene information intact, but more the overall expression structure of the reference.

Is that correct?

**Comment by lazappi** [comment]
Yep, that's the idea for the Splat simulation at least. Some of the other simulations work a bit differentially, for example the Lun2 simulation samples a lot of information from the real dataset instead of fitting a distribution and generating it from that.

**Comment by Hoohm** [comment]
Thanks, that's very helpful!
I'll close the issue.
